content/videos/challenges/54.1-islamic-star-patterns/showcase/contribution3.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "'Watchable' Islamic Star Patterns",
+  "url": "https://editor.p5js.org/mortuish/present/VY-zqiIAp",
+  "author": {
+      "name": "Ya\u011f\u0131z \u0130pek\u00e7i",
```

Comment:
Does this render properly on the site?

Reply:
Yes, there is an option for four different patterns.  Not sure why "watchable" is in the title as they seem static.

Reply:
I checked and it does! But maybe it's best to have the characters directly in the JSON since it's supported.

<img width="464" alt="Screen Shot 2022-07-10 at 9 33 38 AM" src="https://user-images.githubusercontent.com/191758/178147146-d2ae243b-f708-4c9b-a16b-a8485770881a.png">